src/less/badge/base/badge.less

```diff
@@ -1,11 +1,12 @@
+@import "src/less/mixins/utility/utility-mixins.less";
+
 .badge {
+    .customBorderColorProperty('badge-background-color');
```

Comment:
quotes are not needed for arguments in less.
`.customBorderColorProperty(badge-background-color);` both works and looks better (with color syntax)

Reply:
Ohhh, thanks @agliga! I'll go through and update all of these in a quick follow up PR.

Reply:
I still see the custom properties in quotes.

Reply:
Oh, I thought I'd need to update the syntax in the mixin - which would break brandon's changes. Let me check...

Reply:
Works fine. Updated. Thanks Andrew.